<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevicesBrowserWidget</class>
 <widget class="QWidget" name="DevicesBrowserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Device to sync:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="DevicesSelector_">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="MountButton_">
       <property name="text">
        <string>Mount</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="SyncTabs_">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Local collection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>1</number>
       </property>
       <item>
        <widget class="QTreeView" name="OurCollection_"/>
       </item>
       <item>
        <widget class="QProgressBar" name="TSProgress_">
         <property name="value">
          <number>24</number>
         </property>
         <property name="format">
          <string>Transcoding: %p%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="UploadProgress_">
         <property name="value">
          <number>24</number>
         </property>
         <property name="format">
          <string>Uploading: %p%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="UploadButton_">
         <property name="text">
          <string>Upload...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="UpOptsTab_">
      <attribute name="title">
       <string>Transcoding options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>1</number>
       </property>
       <item>
        <widget class="LeechCraft::LMP::TranscodingParamsWidget" name="TranscodingOpts_" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Upload log</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="margin">
        <number>1</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="UploadLog_">
         <property name="html">
          <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LeechCraft::LMP::TranscodingParamsWidget</class>
   <extends>QWidget</extends>
   <header>sync/transcodingparamswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
